src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -212,11 +217,12 @@
                          :disabled? (> page-num (-> library-schema :wizard count))
                          :style {:width 80}}]
           flex/spring
-          (let [save-permissions (or editable? can-share?)
+          (let [save-permissions (or editable? set-discoverable?)
```

Comment:
this doesn't change the outcome of the check - just more readable because these are the 2 different ways/permissions to save data